src/DependencyInjection/SonataAdminExtension.php

```diff
@@ -61,6 +61,11 @@ public function load(array $configs, ContainerBuilder $container): void
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('twig.xml');
         $loader->load('twig_string.xml');
+
+        if (isset($bundles['SonataExporterBundle'])) {
```

Comment:
Can we load config file by alfabhrtic order and after then check bundles? It is easier to read.

Reply:
Ok done. Alphabetical order for required config then optional config